Project1/Problem1.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Problems
+{
+    public class Problem1
+    {
+        public long? SumOfEvenNumbers(int[] numbers)
+        {
+            long? sum = null;
+            if (numbers.Length != 0)
+            {
+                sum = 0;
```

Comment:
Why did you initialized the sum with null? Could the sum to be null?

Reply:
Adding a clarification to scenario 1:
When there are no numbers _in the given array (N=0), there can be_ no sum, resulting in sum = null

Reply:
That is indeed the better approach. There are compelling mathematical arguments for the "empty sum is zero" convention, and so, I will modify the solution and tests accordingly.